quadtree/src/lib.rs

```diff
@@ -217,9 +217,10 @@ impl FromStr for NodeId {
     }
 }
 
+pub const NODE_PREFIX: char = 'r';
 impl fmt::Display for NodeId {
     fn fmt(&self, formatter: &mut fmt::Formatter) -> fmt::Result {
-        formatter.write_char('r')?;
+        formatter.write_char(NODE_PREFIX)?;
```

Comment:
What do you think about changing this to
```rust
pub const NODE_PREFIX: &str = "r";
...
        formatter.write_str(NODE_PREFIX)?;
```
then we wouldn't need `&NODE_PREFIX.to_string()` in `utils.rs`?

Reply:
done :+1: